lms/djangoapps/ccx/migrations/0003_add_master_course_staff_in_ccx.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from ccx_keys.locator import CCXLocator
+from courseware.courses import get_course_by_id
+from contextlib import contextmanager
+from django.db import migrations
+
+from lms.djangoapps.instructor.enrollment import (
+    enroll_email,
+    get_email_params,
+    unenroll_email,
+)
+from lms.djangoapps.instructor.access import (
+    allow_access,
+    list_with_level,
+    revoke_access,
+)
+
+
+@contextmanager
+def ccx_course(ccx_locator):
+    """Create a context in which the course identified by course_locator exists
+    """
+    course = get_course_by_id(ccx_locator)
+    yield course
+
+
+def add_master_course_staff_to_ccx_for_existing_ccx(apps, schema_editor):
+    """
+    Add all staff and admin of master course to respective CCX(s).
```

Comment:
Please add documentation for the arguments:
https://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html